genesis_core/user_api/iam/api/controllers.py

```diff
@@ -44,6 +47,33 @@ def enforce(self, rule, do_raise=False, exc=None):
         return iam.enforcer.enforce(rule, do_raise, exc)
 
 
+class ValidateMixin:
+    vm_min_length = 8
+    vm_not_contain: str = string.whitespace
+    vm_must_contain: list[str] = None  # [digits, ascii_uppercase, punctuation]
```

Comment:
If `vm_must_contain` is a list, maybe `vm_not_contain` should also be a list? For consistency?
Also, these `vm_` prefixes are not necessary.